terraform/modules/prometheus/templates/prometheus.yml.tmpl

```diff
@@ -0,0 +1,17 @@
+global:
+  scrape_interval: 10s
+
+scrape_configs:
+- job_name: teaching-vacancies-prometheus-exporter-dfe-teacher-services
```

Comment:
can this be passed from parmeters as well? keeping the template generic

Reply:
done, thanks